boa/src/exec/mod.rs

```diff
@@ -510,6 +512,7 @@ impl Executor for Interpreter {
                     ValueData::Function(_) => "function",
                 }))
             }
```

Comment:
The reason the interpreter paniced is because the StatementList was not implemented.
Now the code `console.log("hello")` should run.
```suggestion
            },
            Node::StatementList(ref list) => {
                for (i, item) in list.iter().enumerate() {
                    if i + 1 == list.len() {
                        return self.run(item);
                    }
                    self.run(item)?;
                }

                Ok(Gc::new(ValueData::Undefined))
            },
```